addons/hr_holidays/static/src/radio_image_field/radio_image_field.js

```diff
@@ -0,0 +1,9 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { RadioField } from "@web/views/fields/radio/radio_field";
+
+class RadioImageField extends RadioField {}
+RadioImageField.template = "hr_holidays.RadioImageField";
+
+registry.category("fields").add("radio_image", RadioImageField);
```

Comment:
Don't change this in this PR, but I think that addons defining views/widgets should prefix the key in the registry with the addon name

Reply:
Maybe worth notifying them (and others doing the same)